code/mob/living/carbon/human/procs/emote.dm

```diff
@@ -68,15 +68,6 @@
 						#ifdef HALLOWEEN
 						spooktober_GH.change_points(src.ckey, 30)
 						#endif
-						var/possumMax = 15
-						for_by_tcl(responsePossum, /obj/critter/opossum)
-							if (!responsePossum.alive)
-								continue
-							if(!IN_RANGE(responsePossum, src, 4))
-								continue
-							if (possumMax-- < 0)
-								break
-							responsePossum.CritterDeath() // startled into playing dead!
```

Comment:
Why not convert this to the mob critter version?

Reply:
There is already a check doing the same thing for mob opossums just under this one